languages/csharp/exercises/concept/constants/.docs/introduction.md

```diff
@@ -0,0 +1,18 @@
+The `const` modifier can be (and generally should be) applied to any field where its value is known at compile time and will not change during the lifetime of the program.
```

Comment:
I like how briefly you've described it. The `const` modifier can also be applied to variables, but I assume you'll add that to the `after.md` file (which makes sense).

Reply:
Done